result/src/test/scala/uscala/result/ResultSpec.scala

```diff
@@ -2,7 +2,7 @@ package uscala.result
 
 import org.specs2.ScalaCheck
 import org.specs2.mutable.Specification
-import uscala.result.Result.{TraversableResult, Fail, Ok}
+import uscala.result.Result._
```

Comment:
As the name of the implicit class is different depending on the Scala version (`TraversableResult` became `IterableResult`), we should not import it directly anymore. This is how users would do it anyway, as they would get the implicit automatically via `Result` itself (implicit lookup looks on the companion object of a class in the lookup chain, as you know).